app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,3 @@
+<div data-go-back-link='true' style='display:none;line-height:3'>
+  <%= link_to "< Go back (show the login option)", 'javascript:;', class: "#{content_column_class} #{label_column_offset_class}", data: { no_sunet: true, target: '[data-no-sunet-content]', content_target: '[data-no-sunet-target]', link: '[data-go-back-link]' } %>
```

Comment:
Better choice for the < character:

<span aria-hidden="true">&lsaquo; </span>

(creates a visible small arrow, but hides it from screen-reader that would try to read "less than")